README.md

```diff
@@ -1 +1,29 @@
-# subspace-cli
\ No newline at end of file
+# Subspace-CLI
+is a great tool that heavily simplifies the terminal workflow of the farming process. \\
+Instead of running a terminal instance for farmer, and running another terminal instance for the node, now you can run a SINGLE terminal instance to farm!
```

Comment:
``` suggestion
 Instead of running a terminal instance for the farmer, and running another terminal instance for the node, now you can run a SINGLE terminal instance to farm!